api/v1alpha1/conversion.go

```diff
@@ -50,16 +51,26 @@ func Convert_v1alpha1_LinodeObjectStorageBucketSpec_To_v1alpha2_LinodeObjectStor
 	// WARNING: in.Cluster requires manual conversion: does not exist in peer-type
 	out.Region = in.Cluster
 	out.CredentialsRef = in.CredentialsRef
-	out.KeyGeneration = in.KeyGeneration
 	out.SecretType = in.SecretType
 	return nil
 }
+func Convert_v1alpha1_LinodeObjectStorageBucketStatus_To_v1alpha2_LinodeObjectStorageBucketStatus(in *LinodeObjectStorageBucketStatus, out *infrastructurev1alpha2.LinodeObjectStorageBucketStatus, s conversion.Scope) error {
+	out.Ready = in.Ready
+	out.FailureMessage = in.FailureMessage
+	out.Conditions = in.Conditions
+	out.Hostname = in.Hostname
+	out.CreationTime = in.CreationTime
+	// WARNING: in.LastKeyGeneration requires manual conversion: does not exist in peer-type
+	// WARNING: in.KeySecretName requires manual conversion: does not exist in peer-type
+	// WARNING: in.AccessKeyRefs requires manual conversion: does not exist in peer-type
+	return nil
+}
 
 func Convert_v1alpha2_LinodeObjectStorageBucketSpec_To_v1alpha1_LinodeObjectStorageBucketSpec(in *infrastructurev1alpha2.LinodeObjectStorageBucketSpec, out *LinodeObjectStorageBucketSpec, s conversion.Scope) error {
 	// WARNING: in.Region requires manual conversion: does not exist in peer-type
 	out.Cluster = in.Region
 	out.CredentialsRef = in.CredentialsRef
-	out.KeyGeneration = in.KeyGeneration
+	out.KeyGeneration = ptr.To(1)
```

Comment:
Would `0` make more sense here? Or rather, if the Object controller is no longer generating keys, is it safe to leave this as `nil`?